projects/novo-examples/src/examples.routes.ts

```diff
@@ -1,410 +1,409 @@
 })
 export class ChipsPage {}
 
-
 @Component({
   selector: 'date-picker-page',
-  template: `<h1>Date and Time Pickers</h1><p>These allow users to easily select a time and date. It comes in a handful of varieties based on the content of the field.</p><h2>Date Picker <a href="https://github.com/bullhorn/novo-elements/blob/master/src/elements/date-picker">(source)</a></h2><p>The calendar picker is used to select a date. It appears in all date picker fields.</p><h5>Full Date Picker</h5><p><code-example example="date-picker"></code-example></p><h2>Time Picker <a href="https://github.com/bullhorn/novo-elements/blob/master/src/elements/time-picker">(source)</a></h2><p>Time pickers come in 12 hour or 24 hour style.</p><h5>Standalone Time Picker</h5><p><code-example example="time-picker"></code-example></p><h5>Range Picker</h5><p><code-example example="date-range"></code-example></p><h5>Date Time Picker</h5><p><code-example example="date-time"></code-example></p><h5>Date Time Input Picker</h5><p><code-example example="date-time-input"></code-example></p><h5>Customizing Week Start</h5><p><code-example example="week-start"></code-example></p><h5>Different Locale</h5><p>TBD</p>`
+  template: `<h1>Date and Time Pickers</h1><p>These allow users to easily select a time and date. It comes in a handful of varieties based on the content of the field.</p><h2>Date Picker <a href="https://github.com/bullhorn/novo-elements/blob/master/src/elements/date-picker">(source)</a></h2><p>The calendar picker is used to select a date. It appears in all date picker fields.<
```

Comment:
What are these removals all about?

Reply:
That's just an auto-linter adding trailing commas